website/addons/wiki/model.py

```diff
@@ -169,7 +169,7 @@ class NodeWikiPage(GuidStoredObject, Commentable):
 
     page_name = fields.StringField(validate=validate_page_name)
     version = fields.IntegerField()
-    date = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow)
+    date = fields.DateTimeField(default=datetime.datetime.utcnow)
```

Comment:
Why was this changed? `auto_now_add` will ensure that `date` is only set on creation, which is what we want.